### ISSUE TITLE
Reading .pcap files

### ISSUE THREAD

**Issue by MohamedRamzy1** [ouster-lidar/ouster_example#172]
Is there any way to read ouster .pcap files from the ros node ?

**Comment by dllu** [comment]
For now, you may be able to convert them to a bag containing packet messages using this unofficial tool: https://github.com/Krishtof-Korda/pcap-to-bag

**Comment by msy22** [comment]
Is @dllu's answer still up-to-date? [PR #239](https://github.com/ouster-lidar/ouster_example/pull/239) seems to add python tools in the `python/` and `ouster_pcap/` folders.

**Comment by dllu** [comment]
Yeah my answer is outdated. The ouster example contains pcap replay tools now. It's also pretty easy to write a python script to do that.
